src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Converters/RequestTelemetryConverter.cs

```diff
@@ -42,9 +43,14 @@ protected override RequestTelemetry CreateTelemetryEntry(LogEvent logEvent, IFor
             var requestTelemetry = new RequestTelemetry(requestName, requestTime, requestDuration, responseStatusCode, isSuccessfulRequest)
             {
                 Id = operationId,
-                Url = url,
+                Url = url
             };
 
+            if (!String.IsNullOrEmpty(operationName))
```

Comment:
Since we have the operation converter, do we still need this? Shouldn't we rely on the request telemetry object here?

Reply:
I had to change the implementation here because we now set a default value in the RequestLogEntry constructor, see https://github.com/arcus-azure/arcus.observability/pull/229/commits/8804c4bea20c511e20149bd502a02fe8d8a65640
I now check if the request method is present in the operation name (in case a custom operation name is supplied), if not it is added.

I am setting the operation on the ```requestTelemetry.Context.Operation.Name```, but I am not sure if this solves your comment "Shouldn't we rely on the request telemetry object here?". I could use some pointers if certain functionality should be implemented differently